modules/util/portal-tools-soy-builder/bnd.bnd

```diff
@@ -10,4 +10,34 @@ Import-Package:\
 	!org.apache.maven.plugin.*,\
 	!org.apache.tools.ant.*,\
 	*
-Main-Class: com.liferay.portal.tools.soy.builder.SoyBuilder
\ No newline at end of file
+Main-Class: com.liferay.portal.tools.soy.builder.SoyBuilder
+-includeresource:\
```

Comment:
Hi @gamerson! Did you add all these classes just for `ArrayUtil` or there's something more?